manager/rdb/manager_rdb.go

```diff
@@ -94,24 +94,64 @@ func (m *RdbManager) FindRequestCandidates(req *Request) (Policies, error) {
 	mp := map[string]bool{}
 	var policies Policies
 	for _, s := range req.Subjects {
-		res, err := m.table.Filter(func(t r.Term) r.Term {
-			return r.Expr(s).Match(t.Field("subjects").Field("compiled"))
-		}).Run(m.session)
-		if err != nil {
-			return nil, errors.WithStack(err)
-		}
-		defer res.Close()
-		var s schema
-		for res.Next(&s) {
-			if _, ok := mp[s.ID]; !ok {
-				p, err := s.getPolicy()
+		filterResultCh := m.filter(func(t r.Term) r.Term {
```

Comment:
Like we discussed, the resources array may contain regex strings so simply checking if it contains the resource is not enough.